velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -289,15 +289,42 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
       }
     };
 
+    // Test dict-over-constant encoding structure on a complex type.
```

Comment:
This comment doesn't seem correct. I think wrappedVector is always dict(flat), no ?

Reply:
Good catch!
To be more accurate, the wrappedVector is dict(array-encoding) here because the original vector is an ArrayVector.